src/tests/Microsoft.PowerFx.Core.Tests/CheckResultTests.cs

```diff
@@ -0,0 +1,277 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.PowerFx.Syntax;
+using Microsoft.PowerFx.Types;
+using Xunit;
+
+namespace Microsoft.PowerFx.Core.Tests
+{
+    public class CheckResultTests
+    {
+        [Fact]
+        public void Ctors()
```

Comment:
>Ctors [](http://example.com/codeflow?start=20&length=5)

We should prefix all tests with CheckResultTests or any equivalent or we end up with dozens of "Ctors" in test explorer